### PR TITLE
Add Missing Items and Microphone to KaiDevrim

### PR DESCRIPTION
**Pull request by KaiDevrim**
## Describe your project in 1-3 sentences
I am adding onto a previous project, I need some more parts for my keyboard and @sohamb117 wants to make a USB-C microphone.

## Hack Club Slack handle
Kai - Penguin-ō

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by KaiDevrim** [comment]
don't review this quite yet as i am deciding between some things and may not end up doing the monitor backlight project

**Comment by exu3** [comment]
> don't review this quite yet as i am deciding between some things and may not end up doing the monitor backlight project

I'm marking this as a draft so it doesn't get reviewed. Feel free to mark it as ready to review when you're ready!

**Comment by KaiDevrim** [comment]
ok actually finished it, @sohamb117 is willing to help me out

File: KaiDevrim.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## Plan
+
+### NFC
+I plan on using a raspberry pi that I already own, along with a pikvm so it acts as a troubleshooting device for my pi. My raspberry pi will use the PiKVM to act as a keyboard and mouse, and will also allow me to remotely monitor the user in case they need help. I will be using a an nfc card with the nfc pi hat to do all the authentication and stuff and everything. 
+
+### Stream Deck
+For the Stream Deck, I will use a mechanical keypad, attach that to my raspberry pi, and use the OSS https://stream-pi.com/ to control my computer, phone, server, lights, all from a click of a button. I will need the SD card switcher so that I can use one pi for multiple projects.
+
+### Soham Microphone
+For the micrphone, Soham plans on justing follow this very simple video, [Building a quality USB-C microphone](https://www.youtube.com/watch?v=LoQu3XXIayc). He also plans on chipping in the rest of his budget to this.
```
Comment (exu3) [inline KaiDevrim.md:25]: Unfortunately, grant funds aren't able to be transferred across different projects. The only case where a project can receive a grant of $250+ is if there are multiple people working on the same project. It looks like Soham is planning to work on a project of his own.

Comment (exu3) [reply KaiDevrim.md:25]: With that said, would you still be interested in pursuing the proposed additions to your project?

Comment (exu3) [reply KaiDevrim.md:25]: If so, please request me for another review, and I'll take a look as soon as I can.

Comment (sohamb117) [reply KaiDevrim.md:25]: Hi, so Kai and I will be working on this together! It's a joint project, but the total price is staying under 250? We're both building our own projects but we're also pooling our extra funds to work on this one together.

**Review by exu3 — APPROVED** [review]
The project idea looks good to me! The @hackclub/bank-ops team will work with you and @sohamb117 on getting the funds through Hack Club Bank.

**Comment by exu3** [comment]
I see this is marked as a draft. When you're ready, can you request a review from the `@hackclub/bank-ops` team or your Bank point-of-contact?

**Comment by KaiDevrim** [comment]
@MelSmith104 @hackclub/bank-ops

**Comment by MelSmith104** [comment]
hey @KaiDevrim and @sohamb117 ! dm'ing you both right now!

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**